src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -129,54 +132,114 @@ internal static class Defaults
             public const int NumberOfIterations = 100;
         }
 
+        /// <summary>
+        /// Gradient boosting decision tree.
+        /// </summary>
+        /// <remarks>
+        /// For details, please see <a href="https://en.wikipedia.org/wiki/Gradient_boosting#Gradient_tree_boosting">here</a>.
```

Comment:
replace with "Gradient_tree_boosting"

Reply:
done

---
In reply to: [263686400](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263686400) [](ancestors = 263686400)